components/Cards/examples/supplemental/CardEditReorderCollectionCell.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  CardEditReorderCollectionCell.swift
+//  CatalogByConvention
+//
+//  Created by Galia Kaufman on 7/6/18.
+//
+//  Card images source:
+//  Material Design Guidelines (https://material.io/design/components/cards.html#card-collections)
```

Comment:
I can't seem to find the asset downloads on that site. Can we chat offline so I can understand how to get the files?

Reply:
New images are from [here](https://drive.google.com/corp/drive/folders/1GTR_b6gfv2o4mD1QyMX29PWd0vWyla9f) and [here](https://drive.google.com/corp/drive/folders/1IlFU0kBNafbxVvV6Q9RMGHPL544WsZ8A)